gapic-generator-java/src/main/java/com/google/api/generator/gapic/model/Service.java

```diff
@@ -52,6 +55,10 @@ public boolean hasDescription() {
     return !Strings.isNullOrEmpty(description());
   }
 
+  public boolean hasApiVersion() {
+    return !Strings.isNullOrEmpty(apiVersion());
+  }
```

Comment:
Do you know if it's possible where the apiVersion is set, but it's set to an empty string?

i.e. the Service proto has `google.api.api_version = ""`. It's not our responsibility to validate, but I'm wondering if it's possible

Reply:
Treating `api_version=""` the same as `api_version=null` sounds good to me.